src/bindings/python/src/pyopenvino/core/properties/properties.cpp

```diff
@@ -169,13 +169,27 @@ void regmodule_properties(py::module m) {
         return ov::device::priorities(value);
     });
 
+    // Special case: ov::device::PCIInfo
+    py::class_<ov::device::PCIInfo, std::shared_ptr<ov::device::PCIInfo>> cls_pciinfo(m_device, "PCIInfo");
+    cls_pciinfo.def(py::init<const uint32_t&, const uint32_t&, const uint32_t&, const uint32_t&>());
+    cls_pciinfo.def_readwrite("domain", &ov::device::PCIInfo::domain);
+    cls_pciinfo.def_readwrite("bus", &ov::device::PCIInfo::bus);
+    cls_pciinfo.def_readwrite("device", &ov::device::PCIInfo::device);
+    cls_pciinfo.def_readwrite("function", &ov::device::PCIInfo::function);
```

Comment:
why do we need `readwrite`? I supposed it's static information from plugin, which can only be read

Reply:
thats correct, sorry, fixed to readonly. thanks!